rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -497,11 +503,12 @@ public void leaveLRA(URI lraId, String body) throws WebApplicationException {
         });
 
         if (asyncTermination[0] && !paths.containsKey(STATUS) && !paths.containsKey(FORGET)) {
-            LRALogger.i18nLogger.error_asyncTerminationBeanMissStatusAndForget(compensatorClass);
-
```

Comment:
We don't normally make formatting changes (removing a line) amongst other changes.